docs/advanced/input_files/input-main.md

```diff
@@ -2112,8 +2123,7 @@ These variables are used to control berry phase and wannier90 interface paramete
 
 - **Type**: String
 - **Description**:
-  phase of Gauss type electric field  
-  amp*cos(2pi*f(t-t0)+phase)exp(-(t-t0)^2/2sigma^2)
+  phase of Gauss type electric fieldamp*cos(2pi*f(t-t0)+phase)exp(-(t-t0)^2/2sigma^2)
```

Comment:
？

Reply:
Please explain each variable in the function

Reply:
The format is corrected, but I'm not aware of the meaning of these variables, since I'm not familiar with TDDFT.